test/test_beltrami.py

```diff
@@ -167,14 +173,16 @@ def test_beltrami_convergence(actx_factory, operator, solution, visualize=False)
                 qbx_order=5,
                 source_ovsmp=4,
                 resolutions=[32, 64, 96, 128],
+                # FIXME: FMM should not be slower!
                 fmm_order=False, fmm_backend=None,
                 radius=radius
                 )
     elif operator.ambient_dim == 3:
         case = eid.SphereTestCase(
                 target_order=5,
-                qbx_order=5,
+                qbx_order=4,
```

Comment:
This is the actual change in here. I tried bumping up the oversampling, but it didn't seem to help much.